tls/s2n_resume.c

```diff
@@ -776,9 +802,17 @@ S2N_RESULT s2n_resume_encrypt_session_ticket(struct s2n_connection *conn, struct
     /* No keys loaded by the user or the keys are either in decrypt-only or expired state */
     RESULT_ENSURE(key != NULL, S2N_ERR_NO_TICKET_ENCRYPT_DECRYPT_KEY);
 
+    /* Generate unique per-ticket key */
```

Comment:
It might be worth being more careful with your phrasing. TLS1.3 has per-ticket keys ;) I'd specify "ticket encryption key" or something.